packages/react-native-aztec/LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
We don't maintain license files in other packages as there is one license for the whole repository, can it be removed?

Reply:
That's right, removed 👍